src/app/(nextjs_migration)/(authenticated)/user/settings/page.tsx

```diff
@@ -152,6 +153,7 @@ export default async function Settings() {
   if (!session || !session.user?.subscriber) {
     return redirect("/");
   }
+  const enabledFlags = await getFlagsEnabledForEmail(session.user.email);
```

Comment:
Question: Why are we using `getFlagsEnabledForEmail` and not `getFlagsEnabledForEveryone` here? I would expect the change to be shown to every user?

Reply:
The former gets all the flags that are enabled for everyone _and_ the current user, so it's strictly additive - and thus safer to use if you have a user that's logged in.

That said, there's some discussion in the other PR about whether it should be this way, so I might need to update this.

Reply:
OK, so it's now no longer additive - it will now only show up if the flag is enabled and either the allowlist is empty, or the user is on the allowlist. I've chosen not to ignore the allowlist, just in case we want to e.g. test this specifically with QA first, but we'll have to make sure to clear the allowlist once we want to enable this for everyone.